### PR TITLE
add method to send a receiver video constraint message

### PR DESCRIPTION
**Pull request by bbaldino**

### CODE REVIEW

File: modules/RTC/BridgeChannel.js [hunk 1]
```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
+        logger.log('sending a ReceiverVideoConstraint message with '
```
Comment (virtuacoplenny) [inline modules/RTC/BridgeChannel.js]: Where would the code be that calls this?

Comment (bbaldino) [inline modules/RTC/BridgeChannel.js]: the way i see it, there would be a UI element (the quality slider) that would call some logic upon changing which would translate the value of the slider into a constraint

Comment (virtuacoplenny) [inline modules/RTC/BridgeChannel.js]: Would you need more code then to make sure the jitsi-meet UI can easily access this method through a public api? I'm not sure if at this point BridgeChannel is well exposed.

Comment (bbaldino) [inline modules/RTC/BridgeChannel.js]: potentially, though i guess i saw that as part of the ui plumbing work (separate from this)

File: modules/RTC/BridgeChannel.js [hunk 2]
```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
```
Comment (gpolitis) [inline modules/RTC/BridgeChannel.js]: Would you mind removing the "receiver" part from the message name? I can't imagine having a `SendVideoConstraintMessage`.

File: modules/RTC/BridgeChannel.js [hunk 3]
```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
```
Comment (gpolitis) [inline modules/RTC/BridgeChannel.js]: s/fram/frame

File: modules/RTC/BridgeChannel.js (hunk 2 above)
Comment (bbaldino) [inline modules/RTC/BridgeChannel.js]: i actually think being explicit that this is a receiver constraint is useful (even if we don't ever plan to have a send video constraint message, i still think it's nice to be more descriptive).  do you feel strongly about removing it?

File: modules/RTC/BridgeChannel.js (hunk 3 above)
Comment (bbaldino) [inline modules/RTC/BridgeChannel.js]: will fix this

File: modules/RTC/BridgeChannel.js (hunk 2 above)
Comment (gpolitis) [inline modules/RTC/BridgeChannel.js]: Kind of strong, unless you feel even stronger about not removing it :) We don't prepend lastN, nor selected endpoint, for instance, and the idea there is exactly the same.

Comment (bbaldino) [inline modules/RTC/BridgeChannel.js]: but for those messages there are no possible ambiguities, are there?  i think 'VideoConstraintMessage' is too vague when, with things around o/a, we have 'constraints' being expressed from both sides.

Comment (bbaldino) [inline modules/RTC/BridgeChannel.js]: i guess i'm not sure why we'd want to add ambiguity here.  the method is for sending a receive-side video constraint, so why not name it as such?

**Pull request merged**